list.py

```diff
@@ -1,10 +1,10 @@
-class Node:
+class _Node:
```

Comment:
Does this guarantee that this class can't be imported from other files?

Reply:
No, the class can still be imported if I specify it explicitly: `from list import _Node`.
But the prefix `_` does not allow importing `_Node` if I use: `from list import *`. I found out about it [здесь](https://stackoverflow.com/questions/2177817/python-how-do-i-disallow-imports-of-a-class-from-a-module).

I assume that in order to accurately guarantee that the `Node` class cannot be imported, it must be an inner `List` class.

Reply:
Having `_` to prevent the class from being imported in case of using `*` is at least something. I guess that's fine, and looks better than a nested class ✔️